src/_community/rules.md

```diff
@@ -143,9 +143,9 @@ community spaces that have one, which is known as "hoisting".
 
 #### Avatars
 
-All avatars must adhere to the rules above. Additionally, avatars depicting objectionable or Not Safe For Work (NSFW)
-content are disallowed, as well as fully-transparent avatars or avatars that appear to be fully-transparent by 
-matching the platform's background colour.
+All avatars must adhere to the rules above. Additionally, avatars depicting objectionable, harmful (ex: knife or firearm) 
+or Not Safe For Work (NSFW) content are disallowed, as well as fully-transparent avatars or avatars that appear to be 
+fully-transparent by matching the platform's background color. This also applies to PluralKit avatars. 
```

Comment:
Id probably use "pluralkit proxy" or "pluralkit member" here instead of just "pluralkit"